bin/vue-build

```diff
@@ -29,12 +29,13 @@ program
   .option('--port [port]', 'Server port')
   .option('--host [host]', 'Server host', 'localhost')
   .option('--prod, --production', 'Build for production')
+  .option('--mount', 'Auto-create app instance from given component')
   .option('-c, --config [file]', 'Use custom config file')
   .option('-w, --webpack [file]', 'Use custom webpack config file')
-  .option('--no-config', 'You do not want to use config file')
-  .option('--no-webpack', 'You do not want to use webpack config file')
+  .option('--disable-config', 'You do not want to use config file')
```

Comment:
changed `--no-config` to `--disable-config`, if we use `--no-config` the n it will default to `true`, have no idea about how to set the default value to `undefined`